x-pack/plugins/siem/public/components/open_timeline/translations.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const CANCEL = i18n.translate('xpack.siem.open.timeline.cancelButton', {
+  defaultMessage: 'Cancel',
+});
+
+export const COLLAPSE = i18n.translate('xpack.siem.open.timeline.collapseButton', {
+  defaultMessage: 'Collapse',
+});
+
+export const DELETE = i18n.translate('xpack.siem.open.timeline.deleteButton', {
+  defaultMessage: 'Delete',
+});
+
+export const DELETE_SELECTED = i18n.translate('xpack.siem.open.timeline.deleteSelectedButton', {
+  defaultMessage: 'Delete Selected',
+});
+
+export const DELETE_WARNING = i18n.translate('xpack.siem.open.timeline.deleteWarningLabel', {
+  defaultMessage: 'You will not be able to recover this timeline or its notes once deleted.',
+});
+
+export const DESCRIPTION = i18n.translate('xpack.siem.open.timeline.descriptionTableHeader', {
+  defaultMessage: 'Description',
+});
+
+export const EXPAND = i18n.translate('xpack.siem.open.timeline.expandButton', {
+  defaultMessage: 'Expand',
+});
+
+export const FAVORITE_SELECTED = i18n.translate('xpack.siem.open.timeline.favoriteSelectedButton', {
+  defaultMessage: 'Favorite Selected',
+});
+
+export const FAVORITES = i18n.translate('xpack.siem.open.timeline.favoritesTooltip', {
+  defaultMessage: 'Favorites',
+});
+
+export const LAST_MODIFIED = i18n.translate('xpack.siem.open.timeline.lastModifiedTableHeader', {
+  defaultMessage: 'Last Modified',
+});
+
+export const MISSING_SAVED_OBJECT_ID = i18n.translate(
+  'xpack.siem.open.timeline.missingSavedObjectIdTooltip',
+  {
+    defaultMessage: 'Missing savedObjectId',
+  }
+);
+
+export const MODIFIED_BY = i18n.translate('xpack.siem.open.timeline.modifiedByTableHeader', {
+  defaultMessage: 'Modified By',
+});
+
+export const NOTES = i18n.translate('xpack.siem.open.timeline.notesTooltip', {
+  defaultMessage: 'Notes',
+});
+
+export const ONLY_FAVORITES = i18n.translate('xpack.siem.open.timeline.onlyFavoritesButtonLabel', {
+  defaultMessage: 'Only Favorites',
+});
+
+export const OPEN_AS_DUPLICATE = i18n.translate('xpack.siem.open.timeline.openAsDuplicateTooltip', {
+  defaultMessage: 'Open as a duplicate timeline',
+});
+
+export const OPEN_TIMELINE = i18n.translate('xpack.siem.open.timeline.openTimelineButton', {
+  defaultMessage: 'Open Timeline...',
+});
+
+export const OPEN_TIMELINE_TITLE = i18n.translate('xpack.siem.open.timeline.openTimelineTitle', {
+  defaultMessage: 'Open Timeline',
+});
+
+export const PINNED_EVENTS = i18n.translate('xpack.siem.open.timeline.pinnedEventsTooltip', {
+  defaultMessage: 'Pinned Events',
+});
+
+export const POSTED = i18n.translate('xpack.siem.open.timeline.postedLabel', {
+  defaultMessage: 'Posted: ',
```

Comment:
I would remove this space you have here, `Posted: ` and instead put the formatting within the HTML if it is required.